src/components/multi-select/test-queries.js

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
While I was upgrading `@testing-library/react` in bigmaven, I realized we also have a lot of helper queries in our features. It would have made upgrading much easier if all the test queries were `findBy*` because of loading issues. 

I think we can take a pro-tip from this as well: if we want features to re-use our components then we need to make it easier to test and change up their tests. I think if we just provide `findBy*`s then they can easily always `await findBy*`. 

What do you think?

Reply:
My plan was to compose these queries into new, `findBy`-based ones for the autocompleter, since the multi-select has no inherent async state. I don't have a strong opinion on this however, so if we think it'll be easier to use `findBy` here than compose these functions later, I can do that.